src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.scss

```
@@ -4,7 +4,7 @@ cd-info-card {
   padding: 0 0.5vw;
 }
 
-::ng-deep .pg-status-popover-wrapper {
+::ng-deep cd-info-card.pg-status-popover-wrapper {
```

Think its better to use the component selector:
```suggestion
::ng-deep cd-health .pg-status-popover-wrapper {
```